examples/lusid/orders/order-blocks/create-custom-property-on-order-block/0.5-call-block.sql

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
This file is "load data" - does it need to be separate from step 1

Reply:
There is a potential issue It could highlight, where the teardown does not reflect immediately despite query success.

Meant to ask if this is expected behaviour